.vscode/cspell.json

```diff
@@ -35,6 +35,7 @@
     "!bahavior",
     "ABFS",
     "ABNF",
+    "adamdebreceni",
```

Comment:
I've mentioned this before, but IMHO cspell corrections for CHANGELOG entries should probably be in the changelog itself rather than in the global cspell.json file.

Reply:
Agree, but let's do it as more of a centralized effort - #4668.